docs/Configuration.md

```diff
@@ -474,7 +485,29 @@ that contains the whole path of the file, as opposed to just the name of the fil
 What's more, the `file.path.resolved` attribute is automatically recognized by the `sumologicexporter`
 and translated to `_sourceName` attribute in Sumo Logic.
 
-For details, see the [Filelog Receiver documentation][filelogreceiver_readme].
+By default, the Filelog receiver watches files starting at their end,
```

Comment:
Perhaps add a doc link to the respective configuration docs? i.e. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver#configuration

Reply:
Or mention that this default behaviour comes from that particular default in filelogreceiver ?

Reply:
Done